INSTALLATION.md

```diff
@@ -1,218 +1,156 @@
+We're listing a few common approaches to get a running instance of mongodb database:-
 
-        cp .env.sample .env
+1. MongoDB Atlas is the easiest way to get a running instance of mongodb database. It is a hosted(remote) mongodb database provided by mongodb itself. If you're a beginner and don't want too much of a hassle setting up the database you should use this approach. Follow the setup guide on official [mongodb atlas docs](https://www.mongodb.com/docs/atlas/getting-started/). Mongodb Atlas is just one of the many hosted database solutions. If you want to use something else please do your own research.
 
-5.  Generate an `ACCESS_TOKEN_SECRET` using the `openessl` command below and copy the result to the `ACCESS_TOKEN_SECRET` section of the `.env` file.
+2. You can install mongodb natively on your system and create/connect to the database. Follow the setup guide on official [mongodb docs](https://www.mongodb.com/docs/manual/administration/install-community/) for your respective operating system.
 
-        openssl rand -hex 32
+3. If you don't want to bloat your system with unnecessary stuff docker is a great way to manage applications. You can set up a mongodb database inside a docker container and manage it as per your will. Follow the setup guide using this [video tutorial](https://www.youtube.com/watch?v=uklyCSKQ1Po).
 
-6.  Generate an `REFRESH_TOKEN_SECRET` using the `openessl` command below and copy the result to the `REFRESH_TOKEN_SECRET` section of the `.env` file.
 
-        openssl rand -hex 32
+Which approach you choose to set up your mongodb database does not matter. What matters is the connection string to that database using which talawa-api can connect to it. Connection string can differ depending on the approach you used to set up the mongodb database. Please read official [mongodb docs](https://www.mongodb.com/docs/manual/reference/connection-string/) on the connection string. Copy/paste this connection string to the variable named `MONGO_DB_URL` in the `.env` file.
 
-7.  Follow the `Setting Up Mongodb Database` section to set up a mongodb database and get a mongodb database connection string. Paste that connection string in `.env` file under the variable named `MONGO_DB_URL`.
 
-8.  Get the google recaptcha secret key from `google-recaptcha-admin` for reCAPTCHA v2 and "I'm not a robot" Checkbox, and copy the key to the `RECAPTCHA_SECRET_KEY` section of the `.env` file.
+# Setting up google and firebase
 
-    **Note**: In domains, fill localhost
+1. Get the google recaptcha secret key from `google-recaptcha-admin` for reCAPTCHA v2 and "I'm not a robot" Checkbox, and copy the key to the variable named `RECAPTCHA_SECRET_KEY` in the `.env` file.
+
+**Note**: In domains, fill localhost
```

Comment:
Please explain why `localhost` and state when this would not be the case. We need the user to feel comfortable in the installation.